f8a_worker/workers/githubstats.py

```diff
@@ -0,0 +1,55 @@
+from bs4 import BeautifulSoup
+import requests
+import logging
+from f8a_worker.base import BaseTask
+
+logger = logging.getLogger(__name__)
+
+class GithubStats(BaseTask):
+    """ Collects statistics by parsing Github Page """
+    _analysis_name = "github_manifest_details"
```

Comment:
This is not really needed.

Reply:
Removed the analysis name, as it has also been removed from tasks.